README.md

```diff
@@ -1,2 +1,41 @@
-# persistent-volume-migrator
-Tool to migrate Kubernetes In-tree and Flex volumes to Ceph-CSI
+# What is persistent-volume-migrator?
+
+The persistent-volume-migrator tool aims to help migrate Flex RBD Volumes to [Ceph-CSI](https://github.com/ceph/ceph-csi) RBD Volumes.
+
+# Getting Started
+**NOTE**: The Migration procedure will come with a downtime; As we need to scale down the applications using the Volumes before migration.
+## Prerequisite
+This guide assumes you have created a Ceph cluster with Flex and [Ceph-CSI](https://github.com/ceph/ceph-csi).
+1) Minimum ceph version should be Ceph-CSI supported
+2) Existing volumes should be dynamically provisioned.
+3) Stop the application pods that are consuming the flex volume(s) that need to be converted.
+4) `ceph-common`packages, for rbd ceph commands.
+5) Ceph-CSI storageClass should be created in which you want to migrate.
+
+## Steps to Migrate
+1) Stop the application pods that are consuming the flex volume(s) that need to be converted.
+2)   Create rbac required to run migrator pod
+     1) `kubectl create -f tests/manifests/rbac.yaml`
+3) Create migrator pod
+   1) `kubectl create -f tests/manifests/toolbox.yaml`
+4) Download the binary
+   1) `wget <link to binary>`
+5) Run command to start migration:
```

Comment:
Let's have the complete steps, then they go to the other section for detailed command line options
```suggestion
5) Run a command to [migrate the PVC(s)](link to section below)
6) Verify the migration succeeded
7) Start the application pods
```